app/models/taxon.rb

```diff
@@ -26,7 +26,30 @@ def tagged_content_count(content_id)
   private
 
   def content_item
-    @content_item ||= Services.content_store.content_item(base_path)
+    @content_item ||= begin
+                        Services.content_store.content_item(base_path)
+                      rescue GdsApi::ContentStore::ItemNotFound
+                        draft_taxon
+                      end
```

Comment:
This indentation is abit funky, no?

Reply:
I think it's okay, it just looks a bit strange because it's being assigned to the content_item variable?

Reply:
Ah ok cool. Is this a Ruby formatting convention?

Reply:
I did it like that to make it clear that the outcome of the whole block is being assigned to the content_item variable. I don't think that it's a regular Ruby formatting convention, but it didn't fail the linter.